templates/user/profile.html

```diff
@@ -79,6 +80,15 @@ <h1 name=username>{{ user.username or 'Insert a username' }}</h1>
   })
   localizeField.addEventListener('change', event => event.target.checked && localize())
 
+  const publishField = document.querySelector('[name=publish]')
```

Comment:
when you create an article it's right
But after that, I was wondering that user could be publish/unpublish article for is profile view